ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -606,6 +610,20 @@ private void controlScriptsCombo_SelectedIndexChanged(object sender, EventArgs e
 
         }
 
+        /// <summary>
+        /// Functionality for when the STOP Telescope button is pressed. Creates a confirmation pop up box to prevent accidental presses of the stop button before sending any stop script.
+        /// </summary>
+        private void stopRT_click(object sender, EventArgs e)
+        {
+            DialogResult result = MessageBox.Show("Are you sure you want to stop the telescope?", "Telescope Stop Confirmation", MessageBoxButtons.YesNo);
+
+            if (result == DialogResult.Yes)
+            {
+                // Run the stop script for the telescope
+                rtController.RadioTelescope.PLCDriver.InterruptMovementAndWaitUntilStopped(true, false);
```

Comment:
It would also be a good idea to log whenever the stop button is clicked. Look at the other `logger` messages and copy their format so that we log when someone clicks the RT Stop button.

- [ ] Added logger message

Reply:
Just double check that the content of the message is correct, but it should now add logger information when the telescope stops